src/_data/sites/really-useful-storage-boxes.yaml

```diff
@@ -0,0 +1,10 @@
+name: Really Useful Storage Boxes
+description: |-
+ - Have a length between 8 and 20 alphanumeric characters (without accents)
+ - Contain at least 1 CAPITAL letter
+ - Contain at least 1 lowercase letter
+ - Contain at least 1 numeric character
+ - Contain at least 1 special character taken from the following list: *$@&()[]{}=#.-!?+/£€%
```

Comment:
Thanks for the contribution!

Nit: can you indent these line two spaces like the other sites please?

Reply:
abso-toot-ely 🎺